web/app/js/components/MetricsTable.jsx

```diff
@@ -14,7 +14,78 @@ import _isNil from 'lodash/isNil';
 import { processedMetricsPropType } from './util/MetricUtils.jsx';
 import { withContext } from './util/AppContext.jsx';
 
-const columnDefinitions = (resource, showNamespaceColumn, PrefixedLink) => {
+const TcpStatColumns = [
+  {
+    title: "Connections",
+    dataIndex: "tcp.openConnections",
+    isNumeric: true,
+    render: d => metricToFormatter["NO_UNIT"](d.tcp.openConnections),
+    sorter: (a, b) => numericSort(a.tcp.openConnections, b.tcp.openConnections)
+  },
+  {
+    title: "Read Bytes",
```

Comment:
consider something to indicate this is bytes over the last time window? `Read Bytes/min`, assuming our time window is hard-coded?

Reply:
yeah, good point!

Reply:
(we could also consider doing Read Bytes / sec and just dividing by the metric window)